corehq/apps/case_search/temp/fix_child_cases.py

```diff
@@ -0,0 +1,85 @@
+def update_script():
+    from corehq.form_processor.models import CommCareCase
+    from corehq.apps.hqcase.case_helper import CaseHelper
+    from corehq.apps.locations.models import SQLLocation
+    import os
+
+    DOMAIN = 'develop'
+    CHILD_CASE_TYPE = 'membre'
+
+    case_ids = CommCareCase.objects.get_case_ids_in_domain(domain=DOMAIN, type=CHILD_CASE_TYPE)
+
+    total_cases = len(case_ids)
+    cases_checked = -1
+
+    error_log_file_path = os.path.expanduser("~/chris_script_error.log")
+    success_log_file_path = os.path.expanduser("~/chris_script_success.log")
+    with open(success_log_file_path, "a") as success_logfile:
```

Comment:
I am not sure how this script addresses the following general concerns with these kind of scripts., (if it needs to address at all, that is). 
- Don't you need a way of pausing and resuming the script. 
- If it fails in midway, is there a way to skip the ones that already have been processed.
- Is it okay to run it multiple times, is it idempotent on the cases that are already processed?
- Is there a way to submit updates in bulk to improve the speed.